torch/__init__.py

```diff
@@ -325,6 +325,12 @@ def manager_path():
 from . import _torch_docs, _tensor_docs, _storage_docs
 del _torch_docs, _tensor_docs, _storage_docs
 
+def check_env_flag(name, default=''):
+    return os.getenv(name, default).upper() in ['ON', '1', 'YES', 'TRUE', 'Y']
+
+# Prototypes overwrites torch builtins with functions!
+if check_env_flag('BUILD_PROTOTYPES'):
```

Comment:
Actually I'm a little worried that BUILD_PROTOTYPES might be a not-uncommon env variable that some users have set

Reply:
maybe BUILD_NESTEDTENSOR is a better idea anyway. And then we might as well make it torch.nestedtensor right away...